Makefile

```diff
@@ -5,6 +5,10 @@ SHELL=/bin/bash
 
 # task group id for downloading evals and logs
 LOGS_TASK_GROUP?=
+# An ID of a Taskcluster task with a Marian model in the artifacts
+MODEL_TASK?=
```

Comment:
Suggestion (blocking): We may need to discuss this outside of a PR review, but the Makefile approach is not scaling well for what we are adding to it.

Makefiles have the following issues:

1. Our implementation is monolithic, with one file having many concerns.
1. Makefiles provide no built in way to list out the commands that are available.
1. The commands do not have a CLI with documentation and argument parsing.
1. There is no way to pass arguments to the underlying scripts other than grafting on environment variables, which is a second API that isn't documented and not parsed nicely.

I've been feeling the friction of this when trying to interact with our tools. I think it's time we address it. We can make our python utilities all the entry point into our scripts, and provide reasonable defaults for the arguments values.

Reply:
As we discussed live, since it's not clear yet what's the best way to move forward with restructuring our tooling and we want to land this code, I created a follow-up issue: https://github.com/mozilla/firefox-translations-training/issues/495.

Reply:
I re-requested review for myself and would like to check things out locally, and read through the implementation one more time.